### PR TITLE
[13.0][FIX] helpdesk_mgmt_crm: Prevent null value in column "notification_type"

### PR DESCRIPTION
**Pull request by victoralmau**
Prevent null value in column "notification_type" if message have notifications (not copied)

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT38675

### CODE REVIEW

**Review by pedrobaeza — COMMENTED** [review]
I'm not sure about this mechanism and if there's going to be again another notification to the followers. Can you be sure about it?

**Comment by victoralmau** [comment]
> I'm not sure about this mechanism and if there's going to be again another notification to the followers. Can you be sure about it?

I have done some changes and improved the tests to validate as much as possible.

File: helpdesk_mgmt_crm/wizard/helpdesk_ticket_create_lead.py [hunk 1]
```diff
@@ -39,9 +39,24 @@ def _prepare_vals(self):
...
+            new_message = message.copy(
+                {
+                    "model": lead._name,
+                    "res_id": lead.id,
+                    # prevent null value in column "notification_type" if message
+                    # have notifications (not copied)
+                    "notified_partner_ids": False,
+                }
+            )
+            for notification in message.notification_ids:
```
Comment (pedrobaeza) [inline helpdesk_mgmt_crm/wizard/helpdesk_ticket_create_lead.py]: Why do you need to do this?

Comment (victoralmau) [reply helpdesk_mgmt_crm/wizard/helpdesk_ticket_create_lead.py]: Do you think it is not necessary to transfer notifications?

Comment (pedrobaeza) [reply helpdesk_mgmt_crm/wizard/helpdesk_ticket_create_lead.py]: I don't think so.

**Review by pedrobaeza — APPROVED** [review]

**Comment by victoralmau** [comment]
Ping @CarlosRoca13

**Comment by pedrobaeza** [comment]
Let's fast-track it for fixing the problem:

/ocabot merge patch

**Review by marcelsavegnago — COMMENTED** [review]
LGTM

**Pull request merged**